src/lib/utils/index.ts

```diff
@@ -0,0 +1 @@
+export { createHttpParams } from './terra-http-params';
```

Comment:
Das muss auch noch in der public-api exportiert werden, sonst kann es nicht verwendet werden